server/README.md

```diff
@@ -20,6 +20,10 @@ The INS server provides a backend RESTful service for INS-node and front-end web
     `http://localhost:9080/set_rssi/5239/23:43:3d:3e:5e:f5/13/5a:4e:44:ff:5a:6e/40`
     `http://localhost:9080/set_rssi/5239/23:43:3d:3e:5e:f5/11.3/5a:4e:44:ff:5a:6e/44.3`
     `http://localhost:9080/set_rssi/5239/23:43:3d:3e:5e:f5/15.6/5a:4e:44:ff:5a:6e/44.4`
+    
+    http://localhost:9080/set_rssi/5239/23:43:3d:3e:5e:f5/-53/23:43:3d:3e:5e:f6/-52/23:43:3d:3e:5e:f7/-20/23:43:3d:3e:5e:f5/-53/23:43:3d:3e:5e:f6/-52/23:43:3d:3e:5e:f7/-20/23:43:3d:3e:5e:f5/-59/23:43:3d:3e:5e:f6/-51/23:43:3d:3e:5e:f7/-22
+     http://localhost:9080/set_rssi/5239/23:43:3d:3e:5e:f5/-43/23:43:3d:3e:5e:f6/-55/23:43:3d:3e:5e:f7/-18
+      http://localhost:9080/set_rssi/5239/23:43:3d:3e:5e:f5/-59/23:43:3d:3e:5e:f6/-51/23:43:3d:3e:5e:f7/-22
```

Comment:
Why do we need these additional lines in the documentation? In any case they should be formatted appropriately.

Reply:
Like I mentioned in the comments before, these are trivial lines I used for debugging the code. Its not a necessary change but helped when I wanted to send three rssi per command for the localization to work.

Reply:
I really think this should be removed! :laughing: